book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
+
+.. code-block:: bash
+
+    $ composer require 'sensio/distribution-bundle'
```

Comment:
you forgot to indent the code block (the above 3 lines should have 4 extra spaces on the left, so they align with the body of the note)